src/Sulu/Bundle/CategoryBundle/Admin/CategoryAdmin.php

```diff
@@ -59,7 +59,7 @@ public function __construct(
 
     public function configureNavigationItems(NavigationItemCollection $navigationItemCollection): void
     {
-        if ($this->securityChecker->hasPermission(static::SECURITY_CONTEXT, PermissionTypes::EDIT)) {
+        if ($this->securityChecker->hasPermission(static::SECURITY_CONTEXT, PermissionTypes::VIEW)) {
```

Comment:
This is expected behaviour of Sulu we are showing navigation points only for EDIT permissions.

This was decided some time ago. To have a difference between I can select `categories` (VIEW) inside pages and I can see the main menu point (EDIT).

Reply:
mmm.... ok.
So, user with VIEW and ADD shouldn't see navigation point? As the user with VIEW and DELETE? 
It's a little bit confusing for users, as for me... but ok, I can revert to this  logic and just keep the fix for toolbar.

Reply:
@alexander-schranz fixed. 
I'm still a little bit puzzled about applications logic.  I can understand the business decision of hiding menu elements if user cannot change anything (edit), but as for me application is not aligned with this logic. For real we have a hierarchy of roles - EDIT requires VIEW to be able to edit something (user with EDIT without VIEW can see the menu item, but cannot see the list), DELETE/ADD requires EDIT to be able to delete/add something (otherwise they don't see  menu item to perform their  actions). On the other side, application allows to set roles separately, without indicating these requirements or automatically setting all required roles. It leads to confusion, so I think  bug reports are going to  appear again.

Anyway, the toolbar actions are fixed, buttons appear according to users  roles, there is no situation when user see the action button and gets "error" after. Please, review.

Reply:
> VIEW and ADD shouldn't see navigation point.

If you want to `ADD` you also need `EDIT` permissions in case of the UI. There are more usecases in roles that People are allowed to `EDIT` things but not allowed to `ADD` new things then the other way around. And that is why we did choose `EDIT` over `ADD`.

> VIEW

This is more a case because Sulu is API based you require a lot of View Permissions even if you should not see that Menu Items. So to not confused people with limited access with a lot of menu points they not require because they require VIEW permissions to use any kind of selections in pages, ... we decided that you atleast need `EDIT` permissions to see the Navigation Item. There where lot of discussion about it but we decided to go that way. At current state we can not longer change it as that would be a major BC break for all application with roles building on top of that logic.